impl/createApi.go

```diff
@@ -157,9 +157,9 @@ func CreateAPI(filePath, namespace, serviceUrl, apiName, version string, isDryRu
 		return err
 	}
 
-	dirPath := path.Join(utils.GetAPKCTLHomeDir(), apiName)
+	dirPath := path.Join(utils.GetAPKCTLHomeDir(), utils.APIProjectsDir, apiName)
 
-	os.Mkdir(dirPath, os.ModePerm)
+	os.MkdirAll(dirPath, os.ModePerm)
```

Comment:
Why `mkdirAll` ? Shouldn't we have to specify the path correctly?

Reply:
`Mkdir` won't work in this instance as we need nested directories to get created. The changes proposed through this PR need the following nested directory format: `target/apis/<api_name>/*`. As per the codebase prior to these changes, `Mkdir` was sufficient to generate the output we wanted.

`MkdirAll` creates a directory named path, along with any necessary parents, and returns nil, or else returns an error.